src/gobu/remote.go

```diff
@@ -6,10 +6,18 @@ import (
 	"net/http"
 	"os"
 	"path/filepath"
+	"runtime"
 )
 
 func download(version string) {
-	local := filepath.Join(envPath, version+".tar.gz")
+
+	extension := ".tar.gz"
```

Comment:
Read from global var set in the 'main' func, or pass it as input to the this func from 'main'.